bin/behat-cleanup.sh

```diff
@@ -4,8 +4,7 @@
 # Delete the Pantheon site environment after the Behat test suite has run.
 ###
 
-terminus whoami > /dev/null
-if [ $? -ne 0 ]; then
+if ! "$(terminus whoami)"; then
```

Comment:
```suggestion
if ! terminus whoami > /dev/null; then
```
optional, this is entirely stylistic but something about the subshell ^ irks me. 🔢